flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingResultPartition.java

```diff
@@ -44,34 +36,13 @@
 	 */
 	IntermediateDataSetID getResultId();
```

Comment:
This interface can be removed after FLINK-13639 (refactor IntermediateResultPartitionID to consist of IntermediateDataSetID and partition index) is done.